pytest-shutil/setup.py

```diff
@@ -21,7 +21,7 @@
                     'execnet',
                     'contextlib2;python_version<"3"',
                     'pytest',
-                    'path; python_version >= "3.5"',
+                    'path>=16.12.0; python_version >= "3.5"',
```

Comment:
please do not pin, it would prevent to upgrade the package internally as we have 16.10

Reply:
changed to pathlib